File: programmers/js/level01/134240.js
// 푸드 파이트 대회
function solution(food) {    
  const answer = food
    .slice(1)
    .map(elem => ~~(elem / 2))
    .reduce((acc, cur, idx) => acc + `${idx + 1}`.repeat(cur), '');
  
  return answer + '0' + [...answer].reverse().join('');
}

File: programmers/java/level01/134240.java
// 푸드 파이트 대회

class Solution {
    public String solution(int[] food) {
        StringBuilder order = new StringBuilder();
        
        for (int i = 1; i < food.length; i++) {
            order.append(String.valueOf(i)
                            .repeat(food[i] / 2));
        }
        
        return order.toString() + "0" + order.reverse().toString();
    }
}